### PR TITLE
CFE-682: IPv6 addresses are now added to policy variable sys.ip_addresses

### PR DESCRIPTION
**Pull request by oleorhagen**
Changelog: Title

Signed-off-by: Ole Petter <ole.orhagen@northern.tech>

### CODE REVIEW

**Comment by cf-bottom** [comment]
Thanks for submitting a PR! Maybe @vpodzime can review this?

File: libenv/unix_iface.c [hunk 1]
```diff
@@ -704,6 +704,9 @@ static void FindV6InterfacesInfo(EvalContext *ctx, Rlist **interfaces, Rlist **h
                     xsnprintf(prefixed_ip, sizeof(prefixed_ip), IPV6_PREFIX"%s", ip->name);
                     EvalContextClassPutHard(ctx, prefixed_ip, "inventory,attribute_name=none,source=agent");
 
+                    // Add IPv6 address to sys.ip_addresses
+                    RlistAppendScalar(ips, ip->name);
```
Comment (olehermanse) [inline libenv/unix_iface.c]: I think it's more correct to put inside the if below (so if the interface is ignored, don't add IP to ip_addresses), but the same could be said for the code above, so I can fix it in a follow-up.

**Comment by olehermanse** [comment]
@cf-bottom please test this in jenkins

**Comment by cf-bottom** [comment]
Alright, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/2787/)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/2787/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-2787/

File: libenv/unix_iface.c (hunk 1 above)
Comment (vpodzime) [reply libenv/unix_iface.c]: Cannot agree more.

**Comment by oleorhagen** [comment]
Updated the changelog, and left the interface ignoration to be handled along with IPv4

**Comment by oleorhagen** [comment]
Travis failure looks unrelated to me, but I'm no expert here

**Comment by olehermanse** [comment]
> Updated the changelog, and left the interface ignoration to be handled along with IPv4

It works correctly for IPv4. The problematic lines above are also for IPv6.

**Pull request merged**